Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/state/returntypes/ComputeDescriptorState.java

```diff
@@ -25,18 +25,21 @@
 public class ComputeDescriptorState implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    private long lastAnomalyTimeStamp;
-    private double lastAnomalyScore;
-    private DiVectorState lastAnomalyAttribution;
+    private long internalTimeStamp;
```

Comment:
Good callout - ComputeDescriptorState is introduced in 3.8.0-SNAPSHOT and did not exist in 3.7. Earlier, all this information was stored (misguidedly, since we did not foresee RCFCast, say) in the ThresholdedRandomCutForestState.